starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/setup/CustomRecyclerViewAdapter.java

```diff
@@ -0,0 +1,41 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2.setup;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import ly.generalassemb.drewmahrt.shoppinglistver2.R;
+
+/**
+ * Created by justinwells on 10/25/16.
+ */
+
+public class CustomRecyclerViewAdapter extends RecyclerView.Adapter<CustomViewHolder>{
```

Comment:
I suggest giving this class a name more specific to your app, like "ShoppingListRecyclerViewAdapter" or "GroceriesRvAdapter" or something less generic